scipy/stats/stats.py

```diff
@@ -3361,8 +3360,8 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
                                              std2**2, nobs2)
     return _ttest_ind_from_stats(mean1, mean2, denom, df)
 
-
-def ttest_ind(a, b, axis=0, equal_var=True):
+    
+def ttest_ind(a, b, axis=0, equal_var=True, permutations=None, random_state=None):
```

Comment:
These changes probably should not be included. One convenient recipe to do that is to do
```
git reset HEAD^
git add -p   # it will ask you whether you want to retain each change, answer y or n
git commit -m "STY: optimize: get rid of pysum"
git checkout scipy/stats/stats.py  # throw away the rest of the changes in the file
```